tests/js/sentry-test/utils.tsx

```diff
@@ -0,0 +1,19 @@
+import {BoundFunctions, FindByText} from 'sentry-test/reactTestingLibrary';
+
+/**
+ * Search for a text broken up by multiple html elements
+ * e.g.: <div>Hello <span>world</span></div>
+ */
+export async function findByTextContent(
+  screen: BoundFunctions<{findByText: FindByText}>,
+  textMatch: string | RegExp
+): Promise<HTMLElement> {
+  return screen.findByText((_, contentNode) => {
+    const hasText = (node: Element) => node.textContent === textMatch;
```

Comment:
Would you be able to use `node.innerText` to search the element and the text of its children?

Reply:
no it has to be `textContent`, otherwise it will always be `undefined`

Reply:
It seems that `innerText` is not implemented in `jsdom`.. see [here](https://github.com/jsdom/jsdom/issues/1245).